torchvision/datasets/pcam.py

```diff
@@ -0,0 +1,136 @@
+import os
+import os.path
+import pathlib
+from typing import Any, Callable, Optional, Tuple
+
+from PIL import Image
+
+from .utils import download_file_from_google_drive, extract_archive, verify_str_arg
+from .vision import VisionDataset
+
+
+class PCAM(VisionDataset):
+    """`PCAM Dataset   <https://github.com/basveeling/pcam>`_.
+
+    The PatchCamelyon dataset is a binray classification dataset with 327,680
+    color images (96 x 96px), extracted from histopathologic scans of lymph node
```

Comment:
Not sure if there is a canonical way to write image sizes

```suggestion
    color images (96px x 96px), extracted from histopathologic scans of lymph node
```